riscemu/priv/ElfLoader.py

```diff
@@ -21,15 +23,19 @@ class ElfBinaryFileLoader(ProgramLoader):
     This loader respects local and global symbols.
     """
 
+    is_binary = True
+
     program: Program
 
-    def __init__(self, source_path: str, options: T_ParserOpts):
-        super().__init__(source_path, options)
+    def __post_init__(self):
         self.program = Program(self.filename)
 
     @classmethod
-    def can_parse(cls, source_path: str) -> float:
-        with open(source_path, "rb") as f:
+    def can_parse(cls, source_name: str) -> float:
```

Comment:
why float?

Reply:
This is basically a "confidence" level, of how confident a loader is that he can load the given file. I implemented it like this back then because I liked these style of "bidding" algorithms back then, but it's a bit useless tbh. Maybe we can change this to a lookup table in another patch, as a lookup would be a lot more debuggable^^